extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -90,19 +91,25 @@ describe('integration test: a4a', () => {
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
     xhrMock = sandbox.stub(Xhr.prototype, 'fetch');
+    // Expect key set fetches for signing services.
+    const fetchJsonMock = sandbox.stub(Xhr.prototype, 'fetchJson');
```

Comment:
I really want a less fragile way to do test doubles for AJAX requests than these mocks. No need to fix that in this PR though, just something for the future.